documentation/docs/how_tos/integration-services-sdk/search-channel-and-validate-data.mdx

```diff
@@ -0,0 +1,64 @@
+---
+image: /img/integration-services/logo/integration_services.png
+description: The example-7 script creates 2 identities, and later uses them to demonstrate how you can detect if data has been tampered with by validating using the Tangle. 
+keywords:
+- how to
+- channels
+- authorize channel
+- subscribe channel
+- read channel
+- validate channel data
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Search Channel and Validate Data
+
+The **example-7** script is similar to previous example ([Authorize Channel](authorize-to-channel)) that creates a
+channel as `Owner` and authorizes a `User` to write on it but:
+
+
+<Tabs groupId={"programmingLanguage"}>
+<TabItem value="Java" label={'Java'}>
+
+```bash
+mvn exec:_java -Dexec.mainClass=net.gradbase.how_tos.SearchChannelAndValidateData
+```
+
+**Example source code**: [Example-7](https://github.com/albydeca/iota-is-sdk/blob/main/examples/src/main/java/net/gradbase/examples/SearchChannelAndValidateData.java)
+
+</TabItem>
+<TabItem value="Nodejs" label={'Node.js'}>
+
+```bash
+npm run example-7
+````
+
+**Example source code**: [Example-7](https://github.com/iotaledger/integration-services/blob/master/clients/client-sdk/examples/5-SearchChannelAndValidateData.ts)
+
+</TabItem>
+</Tabs>
+
+## Validate Channel's Data
+
+Since the signature does not match, `results` is a set of tampered data, so the validation will notify it when running the following script:
```

Comment:
```suggestion
Since the signature does not match, `results` is a set of tampered data, so the validation will notify it when running the following script:
```
"It" meaning the signature, a set of tampered data, or there should be something "will notify you about it"?